junit-runtime/src/main/scala/org/junit/Assert.scala

```diff
@@ -291,34 +291,37 @@ object Assert {
   def assertThat[T](reason: String, actual: T, matcher: Matcher[T]): Unit =
     MatcherAssert.assertThat(reason, actual, matcher)
 
-  def assertThrows(expectedThrowable: Class[_ <: Throwable],
-      runnable: ThrowingRunnable): Unit = {
-    expectThrows(expectedThrowable, runnable)
-  }
-
-  def expectThrows[T <: Throwable](expectedThrowable: Class[T], runnable: ThrowingRunnable): T = {
-    try {
-      runnable.run()
-    } catch {
-      case actualThrown: Throwable =>
-        if (expectedThrowable.isInstance(actualThrown)) {
-          actualThrown.asInstanceOf[T]
-        } else {
-          val mismatchMessage = format("unexpected exception type thrown;",
-            expectedThrowable.getSimpleName, actualThrown.getClass.getSimpleName)
-
-          val assertionError = new AssertionError(mismatchMessage)
-          assertionError.initCause(actualThrown)
-          throw assertionError
-        }
-    }
-    val message =
-      s"expected ${expectedThrowable.getSimpleName} to be thrown," +
-      " but nothing was thrown"
-    throw new AssertionError(message)
-  }
-
-  trait ThrowingRunnable {
-    def run(): Unit
-  }
+  // The following methods will be available on JUnit 4.13, a dummy implementation
+  // is being tested in JUnitAssertionTest until 4.13 is released.
+
+  //  def assertThrows(expectedThrowable: Class[_ <: Throwable],
+  //      runnable: ThrowingRunnable): Unit = {
+  //    expectThrows(expectedThrowable, runnable)
+  //  }
+  //
+  //  def expectThrows[T <: Throwable](expectedThrowable: Class[T], runnable: ThrowingRunnable): T = {
+  //    try {
+  //      runnable.run()
+  //      val message =
+  //        s"expected ${expectedThrowable.getSimpleName} to be thrown," +
+  //        " but nothing was thrown"
+  //      throw new AssertionError(message)
+  //    } catch {
+  //      case actualThrown: Throwable =>
+  //        if (expectedThrowable.isInstance(actualThrown)) {
+  //          actualThrown.asInstanceOf[T]
+  //        } else {
+  //          val mismatchMessage = format("unexpected exception type thrown;",
+  //            expectedThrowable.getSimpleName, actualThrown.getClass.getSimpleName)
+  //
+  //          val assertionError = new AssertionError(mismatchMessage)
+  //          assertionError.initCause(actualThrown)
+  //          throw assertionError
+  //        }
+  //    }
+  //  }
+  //
+  //  trait ThrowingRunnable {
+  //    def run(): Unit
+  //  }
```

Comment:
I suggest using `/*` at the beginning and `*/` at the end, on separate lines, so that not all of these lines are affected by a diff. *Especially* since the actual fix that is in there is therefore completely invisible in the diff.